.changeset/serious-ducks-check.md

```diff
@@ -0,0 +1,5 @@
+---
+'@guardian/support-dotcom-components': patch
+---
+
+prefer-nullish-coalescing lint rule violations fix
```

Comment:
There is one file that DCR uses but it can wait for the next release - I don't think there's any need to deploy it at this point.  The changeset is still useful though.